external-scripts.json

```diff
@@ -1,10 +1,4 @@
 [
-  "hubot-google-translate",
-  "hubot-maps",
-  "hubot-google",
   "hubot-postgres-brain",
-  "hubot-better-voting",
-  "hubot-google-images",
-  "hubot-youtube",
```

Comment:
Youtube is used fairly frequently in #music, and occasionally in other channels

Reply:
Done.